### DISCUSSION TITLE
最新版的网络搜索功能不能用了吗？

### DISCUSSION THREAD

**Discussion by linzez** [wenda-LLM/wenda#309] [Q&A]
选  sgwx知识库全文爬取 没有搜索网页内容。结果只搜索了本地那几个txt。和两个网页404代码：

精炼地总结以下文段中与问题相关的信息为二十个字。

    <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
    <html>
        <head>
            <title>Error: 404 Not Found</title>
            <style type="text/css">
              html {background-color: #eee; font-family: sans;}
              body {background-color: #fff; border: 1px solid #ddd;
                    padding: 15px; margin: 15px;}
              pre {background-color: #eee; border: 1px solid #ddd; padding: 5px;}
            </style>
        </head>
        <body>
            <h1>Error: 404 Not Found</h1>
            <p>Sorry, the requested URL <tt>&#039;http://127.0.0.1:17860/api/read_sgwx&#039;</tt>
               caused an error:</p>
            <pre>Not found: &#039;/api/read_sgwx&#039;</pre>
        </body>
    </html>

**Comment by l15y** [comment]
没开配置吧。
新版默认配置中关闭了在线知识库，因为离线模式能力已远超在线模式

**Reply by linzez** [reply]
原来是这一段的sosowx是sogowx才对，你写错一个字母了，复制了这个用不了。
     #sosowx      sogo微信公众号搜索，可配合相应auto实现全文内容分析

还有老大画图提示下面这个，已经打开api和改成786了。能不能下次更新手册里面上一下教程
连接SD API失败，请确认已开启agents库，并将SD API地址设置为127.0.0.1:786

**Comment by HHUtian** [comment]
> 没开配置吧。 新版默认配置中关闭了在线知识库，因为离线模式能力已远超在线模式

大佬，应该怎么打开了，能否说一下，不动代码 ，望请理解，如果能针对新版本出个详细的教程就更好了，谢谢
